bc_obps/registration/models/facility.py

```diff
@@ -35,6 +41,19 @@ class Types(models.TextChoices):
         blank=True,
         null=True,
     )
```

Comment:
I had originally taken out bcghg id and swrs facility id because they're not in the form, but bcghg is in the grid (talking to Zoey about it) so I'm going to leave them both in for now